index.html

```diff
@@ -20,15 +20,17 @@ <h2>Software Engineering Student</h2>
   <div class="column">
 
 <!-- this article contains the "intro" section -->
-<article class="center wrapper space">
+<article class="center wrapper">
 <!-- Headshot -->
-<img src="https://gdurl.com/B2HM" class="round" alt="Jenna's Headshot" width=500>
+<div class="item">
+<img src="https://gdurl.com/B2HM" class="round" alt="Jenna's Headshot">
 
 <p class="special">CREATIVE CALCULATIONS</p>
```

Comment:
why won't this stay aligned with the other parts of the page when width becomes smaller?